drivers/display/display_st7789v.c

```diff
@@ -254,11 +255,48 @@ static int st7789v_set_pixel_format(const struct device *dev,
 static int st7789v_set_orientation(const struct device *dev,
 			    const enum display_orientation orientation)
 {
-	if (orientation == DISPLAY_ORIENTATION_NORMAL) {
-		return 0;
+	struct st7789v_data *data = (struct st7789v_data *)dev->data;
+	uint8_t tx_data = 0;
+	uint16_t x_offset = 0;
+	uint16_t y_offset = 0;
+
+	switch (orientation)
+	{
+	case DISPLAY_ORIENTATION_NORMAL:
+		tx_data |= ST7789V_MADCTL_MV_NORMAL_MODE;
+		x_offset = data->x_offset;
+		y_offset = data->y_offset;
+		break;
+
+	case DISPLAY_ORIENTATION_ROTATED_90:
```

Comment:
My display is rotated counter clockwise 90 degrees. It is not documented which direction the rotation is expected. If feels to me more logical to do clockwise. Do you know what is the correct direction?